### PR TITLE
Add feature flag listing 

### PR DESCRIPTION
**Pull request by xsadia**

### CODE REVIEW

File: pkg/models/organization_model.go [hunk 1]
```diff
@@ -35,20 +36,46 @@ func (om *OrganizationModel) FindByID(ctx context.Context, id primitive.ObjectID
...
 
 	if !ok {
-		return primitive.ObjectID{}, errors.New("unable to assert type of objectID")
+		return primitive.NilObjectID, errors.New("unable to assert type of objectID")
 	}
 
 	return objectID, nil
 }
 
+func (om *OrganizationModel) UserHasReadPermission(
```
Comment (xsadia) [inline pkg/models/organization_model.go:54]: I don`t think this should be here right @LuisOw

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # AUTOMATIC REVIEWERS
 @xsadia
-@LuisOw
\ No newline at end of file
+@luisow
```
Comment (LuisOw) [inline .github/CODEOWNERS:3]: :(

File: pkg/models/organization_model.go (hunk 1 above)
Comment (LuisOw) [reply pkg/models/organization_model.go:54]: Then move it to ``api_utils.go`` or create a new helper file.

Comment (LuisOw) [reply pkg/models/organization_model.go:54]: Actually, add the ```if !isMember {
		return apiutils.ErrReadPermissionDenied
	}
```
to the similar function on ``feature_flag_modelgo`` and move that to another file

**Review by LuisOw — APPROVED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (xsadia) [reply .github/CODEOWNERS:3]: I'll remove you from the org

**Review by LuisOw — APPROVED** [review]

**Pull request merged**